src/Features/LanguageServer/Protocol/Handler/ISolutionRequiredHandler.cs

```diff
@@ -0,0 +1,10 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.CodeAnalysis.LanguageServer.Handler;
+
+internal interface ISolutionRequiredHandler
```

Comment:
I don't like this name, because something can implement it by not require a solution, but I don't know of an alternative, and it can wait until API review